### PR TITLE
Adding the show and hide timeout ids as properties on the tipsy objec…

### PR DESCRIPTION
**Pull request by CliftonH**
…t so that they can be cancelled when destroying the instance. Fixing some small syntax errors.

### CODE REVIEW

**Comment by gilmoreorless** [comment]
All the formatting changes in the diff make it very difficult to see what functionality has been changed.

This also overwrites the changes done by PR #4

File: src/javascripts/jquery.tipsy.js [hunk 1]
```diff
@@ -42,16 +43,18 @@
             var title = this.getTitle();
             if (title && this.enabled) {
                 var $tip = this.tip();
-                
+
                 $tip.find('.tipsy-inner')[this.options.html ? 'html' : 'text'](title);
                 $tip[0].className = 'tipsy'; // reset classname in case of dynamic gravity
-                $tip.remove().css({top: 0, left: 0, visibility: 'hidden', display: 'block'}).appendTo(document.body);
+                $tip.remove().css({top: 0, left: 0, visibility: 'hidden', display: 'block'}).prependTo(document.body);
```
Comment (bqx) [inline src/javascripts/jquery.tipsy.js]: Whats the reason that this was changed from `appendTo` to `prependTo`?

File: src/javascripts/jquery.tipsy.js [hunk 2]
```diff
@@ -70,30 +73,42 @@
                 }
 
                 var pos = $.extend({}, this.$element.offset(), {
-                    width: this.$element[0].getBoundingClientRect().width,
-                    height: this.$element[0].getBoundingClientRect().height
+                    width: this.$element[0].offsetWidth,
+                    height: this.$element[0].offsetHeight
```
Comment (bqx) [inline src/javascripts/jquery.tipsy.js]: This overwrites the changes from PR #4 , could you update this to take that into account?

File: src/javascripts/jquery.tipsy.js [hunk 3]
```diff
@@ -217,36 +246,47 @@
...
+
+        if (!options.live) this.each(function () {
+            get(this);
+        });
+
         if (options.trigger != 'manual') {
-            var eventIn  = options.trigger == 'hover' ? 'mouseenter.tipsy focus.tipsy' : 'focus.tipsy',
-                eventOut = options.trigger == 'hover' ? 'mouseleave.tipsy blur.tipsy' : 'blur.tipsy';
+            var eventIn = options.trigger == 'hover' ? 'mouseenter.tipsy' : 'focus.tipsy',
+                eventOut = options.trigger == 'hover' ? 'mouseleave.tipsy' : 'blur.tipsy';
```
Comment (bqx) [inline src/javascripts/jquery.tipsy.js]: Is this change related to the show and hide timeouts?

**Comment by chrisdarroch** [comment]
Closed because of major formatting changes and general inactivity.
